config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+JMtK6QR7LcqM+VAd3CrEUUiK0I19m1G+YuauRQ7KP0zso1WgGlYGQ4mhTTcLOvrTN/FI6i+sxtv5sm8CfKcweTmhcOBXm4fWzTBHgAGsYAkVgiLoNOZd/1WPy/bCL9+FqEPp3yban3mTNmKneqEyTI/1x+1VCpIOn5bzLflFhd0f5Sf6mfhZ26HiyRPfrFQNV0M8uN+rHfCu2o8mUIkqhEP887dHXYk3cgOh06uLZv9gttvAva+HWX8+8yc2gLKgCA2hpvodg8k9tmQG5uZp09rTUFc4WvMy3X2hTSWpanm+kh3CE18c7Q8jgHwzYPB45FkRJuMszDQRhWaj3s9bUSARX64vi/oqcY5IJpynVFygbWUwRrCemJRXhrX21cb0h+IuMke1OIL52hxG5QpVEKS3mRQ72RsP/i1E--MQwS3/pBMhisEsGy--DFfKINhbvGwP3t8jW3d0WQ==
```

Comment:
this feels not secure

Reply:
I think as long as the master.key file isn't published to version control, this is fine, but also not an expert :D https://edgeguides.rubyonrails.org/security.html#custom-credentials